src/app/shared/components/product/product-reviews/product-reviews.component.html

```diff
@@ -0,0 +1,24 @@
+<h3>{{ 'product.reviews.section.heading' | translate | uppercase }}</h3>
+
+<div *ngIf="(productReviews$ | async)?.length > 0; else noReviews">
+  <div *ngFor="let review of productReviews$ | async">
+    <div class="row pb-3">
+      <div class="col-md-3 text-muted small">
+        {{ review.authorFirstName }}
+        <br />
+        {{ review.creationDate | date: 'MMM d, y, h:mm:ss a' }}
+      </div>
+      <div class="col-md-9">
+        <ng-container *ngFor="let fill of stars$ | async; let lastStar = last">
+          <ish-product-rating-star [filled]="fill" [lastStar]="lastStar"></ish-product-rating-star>
+          <span *ngIf="lastStar" class="mb-0 h5 font-weight-light"> {{ review.title }}</span>
```

Comment:
use class 'review-item-header-title' here - define further styles in the ratings.scss if needed